homeassistant/components/pvpc_hourly_pricing/sensor.py

```diff
@@ -51,9 +51,9 @@ async def async_setup_entry(
 class ElecPriceSensor(RestoreEntity, SensorEntity):
     """Class to hold the prices of electricity as a sensor."""
 
-    unit_of_measurement = UNIT
-    icon = ICON
-    should_poll = False
+    _attr_native_unit_of_measurement = UNIT
```

Comment:
@emontnemery, Perfect! this change was in my TODO list 😅

it would be great to also add **`_attr_state_class = STATE_CLASS_MEASUREMENT`**, which is needed here (now users are tweaking it in Customizations) 🙏🥺

Reply:
Sure thing, fixed in a796715